vsg/rules/align_tokens_in_region_between_tokens_unless_between_tokens.py

```diff
@@ -142,6 +145,19 @@ def analyze(self, oFile):
                            dAnalysis = {}
 
                    if self.case_control_statements_ends_group:
+                       if check_for_case_keywords(iIndex + 1, lTokens) or check_for_when_keywords(iIndex + 1, lTokens):
+                           add_adjustments_to_dAnalysis(dAnalysis, self.compact_alignment)
+
+                           for iKey in list(dAnalysis.keys()):
+                               if dAnalysis[iKey]['adjust'] != 0:
+                                   oLineTokens = oFile.get_tokens_from_line(iKey)
+                                   sSolution = 'Move ' + dAnalysis[iKey]['token_value'] + ' ' + str(dAnalysis[iKey]['adjust']) + ' columns'
+                                   oViolation = violation.New(oLineTokens.get_line_number(), oLineTokens, sSolution)
+                                   oViolation.set_action(dAnalysis[iKey])
+                                   self.add_violation(oViolation)
+
+                           dAnalysis = {}
+                   elif self.case_keyword_statements_ends_group:
```

Comment:
We should be able to delete lines 160 through 172 since we would be removing `self.case_keyword_statements_ends_group`

Reply:
Done